src/version.h

```diff
@@ -1 +1 @@
-#define VERSION_STRING "dev"
+#define VERSION_STRING "v0.0.1"
```

Comment:
```suggestion
#define VERSION_STRING "v0.0.0"
```
This will be overridden during dev and prod builds anyway, it only matters for local builds.